spec/controllers/api/v1/status_messages_controller_spec.rb

```diff
@@ -2,65 +2,91 @@
 
 describe Api::V1::StatusMessagesController do
   describe 'POST #create' do
-    context 'when it\'s the first status and status is empty' do
-      let(:status) { FactoryGirl.attributes_for(:status_message_without_status) }
 
-      it 'does not save the new status' do
-        expect {
-          post :create, status_message: status
-        }.not_to change{ StatusMessage.count }
+    context 'with an invalid token' do
+      before do
+        @user = FactoryGirl.create(:user)
+        request.env['HTTP_AUTHORIZATION'] = "Token token='XYZ'"
```

Comment:
@anymoto minor thing, but during our pairing we learned that single quotes didn't work (they get interpreted as part of the api token itself, `'XYZ'` instead of `XYZ`) For consistency, can we remove them here?